### ISSUE TITLE
Report offenses, ignoring source code disable comments

### ISSUE THREAD

**Issue by khiga8** [rubocop/rubocop#11655]
Is there an existing config we can set when running rubocop that reports all offenses, ignoring any [source code comments](https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code) that might be in place to inline disable a rule? I might have missed it but didn't see it in: [Basic usage](https://docs.rubocop.org/rubocop/usage/basic_usage.html)

**Comment by jonas054** [comment]
Yes there is. It's the `---ignore-disable-comments` option, which you can get from `ruboco --help`, but which didn't make it onto the Basic usage page. I guess we'll have to assume that someone made the decision at some point that it was an advanced option and didn't list it with the basic ones.

**Comment by dvandersluis** [comment]
@khiga8 @jonas054 the option is documented now and will show up on the Basic Usage page once the next version of RuboCop is released.
